dev/VSIX/ProjectTemplates/Desktop/CSharp/PackagedApp/WapProj/Package-managed.appxmanifest

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="$guid9$"
+    Publisher="$XmlEscapedPublisherDistinguishedName$"
+    Version="1.0.0.0" />
+
+  <Properties>
+    <DisplayName>$projectname$</DisplayName>
+    <PublisherDisplayName>$XmlEscapedPublisher$</PublisherDisplayName>
+    <Logo>Images\StoreLogo.png</Logo>
+  </Properties>
+
+  <Dependencies>
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.0.0" />
+    <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.14393.0" MaxVersionTested="10.0.14393.0" />
```

Comment:
Are these real values? Min=Min=10.0.0.0 won't actually used, right?

Shouldn't MinVersion=10.0.17763.0? That's Redstore 5 = October 2018 Update. Project Reunion's only targeting down to RS5.

10.0.14393.0 = Redstore1 = Anniversary Update = August 2, 2016

And shouldn't MaxVersion be something more current? 10.0.19042.0 (20H2) or something?

See [here](https://en.wikipedia.org/wiki/Windows_10_version_history#Channels0) for version info

Reply:
Good catch - thought I'd caught all WinUI references to 17134, 14393, etc.  I *think* Min=Max=10.0 simply uses the latest SDK installed.  But the templates should at least be consistent, and some use specific versions for Windows.Universal.  Will rationalize them all to Min=17763 (RS5) and Max=19042.